File: packages/java/src/main/java/com/tesote/sdk/errors/TlsException.java
package com.tesote.sdk.errors;

public final class TlsException extends TransportException {
    public TlsException(String message, RequestSummary requestSummary,
                        int attempts, Throwable cause) {
        super(message, "TLS_ERROR", 0, null, null, null, null,
                requestSummary, attempts, cause);
    }
}

File: packages/csharp/src/Errors/TlsException.cs
using System;

namespace Tesote.Sdk.Errors;

/// <summary>Certificate or TLS handshake failure.</summary>
public sealed class TlsException : TransportException
{
    /// <summary>Construct from a <see cref="RequestSummary"/> and underlying cause.</summary>
    public TlsException(string? message, RequestSummary? requestSummary, int attempts, Exception? cause)
        : base(message, "TLS_ERROR", 0, null, null, null, null, requestSummary, attempts, cause)
    {
    }
}
